apps/service-catalog/middleware.ts

```diff
@@ -24,6 +24,16 @@ export default withAuth(async function middleware(req: NextRequestWithAuth) {
     `/catalogs/${catalogId}/services/new`,
   ];
 
+  //CatalogId does not have correct format
+  if (catalogId && !validOrganizationNumber(catalogId)) {
+    return NextResponse.rewrite(new URL('/not-found/', req.url));
+  }
+
+  // ServiceId does not have correct format
+  if (serviceId && !validUUID(serviceId)) {
+    return NextResponse.rewrite(new URL('/not-found/', req.url));
+  }
+
   // User do not have read permission in the catalog
```

Comment:
Fjerne kommentar (?)

Reply:
De er egentlig lagt til med vilje for å forklare hva middleware'en gjør 😅 Men kan ta de bort vist du tenker at det er forståelig nok fra før

Reply:
Tror vi har de samme sjekkene veldig mange steder, så tenker de er rimelig godt kjent for de som jobber i repoet:)

Reply:
Og så forklarer koden seg selv godt her syns jeg:)